runtime/doc/builtin.txt

```diff
@@ -4315,19 +4319,17 @@ getregion({pos1}, {pos2} [, {opts}])			*getregion()*
 		- If the selection starts or ends in the middle of a multibyte
 		  character, it is not included but its selected part is
 		  substituted with spaces.
-		- If {pos1} or {pos2} equals "v" (see |line()|) and it is not in
-		  |visual-mode|, an empty list is returned.
-		- If {pos1}, {pos2} or {type} is an invalid string, an empty
-		  list is returned.
-		- If {pos1} or {pos2} is a mark in different buffer, an empty
-		  list is returned.
+		- If {type} is an invalid string, an empty list is returned.
+		- If {pos1} or {pos2} is not current buffer, an empty list is
+		  returned.
 
 		Examples: >
 			:xnoremap <CR>
-			\ <Cmd>echow getregion('v', '.', #{ type: mode() })<CR>
+			\ <Cmd>echow getregion(
+			\ getpos('v'), getpos('.'), #{ type: mode() })<CR>
 <
 		Can also be used as a |method|: >
-			'.'->getregion("'a")
+			'.'->getregion(getpos("'a"))
```

Comment:
In this example '.' is a string representing current position? But pos1 couldn't be a string, right?

Reply:
Fixed.